src/views/v2/emails/admin-failed-order.php

```diff
@@ -16,8 +16,16 @@
  * @var Tribe_Template  $this  Current template object.
  */
 
+ // @todo @codingmusician @juanfra Replace hardcoded data with dynamic data.
+
 $this->template( 'template-parts/header' );
 
-$this->template( 'template-parts/body' );
+$this->template( 'template-parts/body/title' );
+$this->template( 'template-parts/body/order/error-message' );
+$this->template( 'template-parts/body/order/purchaser-details' );
+$this->template( 'template-parts/body/order/event-title' );
+$this->template( 'template-parts/body/order/ticket-totals' );
+$this->template( 'template-parts/body/order/order-total', [ 'status' => 'failed' ] );
+$this->template( 'template-parts/body/order/payment-info' );
```

Comment:
The idea of the `template-parts` folder is that it will be the home for template parts that are used across templates (header, footer).

Then each template will have its own folder where we should be having the templates (atomized) for the different email templates.

For example, we'll be having folders for:

- `v2/tickets`
- `v2/rsvp`
- `v2/failed-order`
- and so forth.

The idea of the main template for each email is to have a structure of:

```
v2/template-parts/header

v2/email-template/body

v2/template-parts/footer
```